tyxe/guides.py

```diff
@@ -114,22 +120,22 @@ def _setup_prototype(self, *args, **kwargs):
             unconstrained_value = biject_to(site["fn"].support).inv(constrained_value)
             if self.train_loc:
                 unconstrained_value = pynn.PyroParam(unconstrained_value)
-            ag.guides._deep_setattr(self, name + ".loc", unconstrained_value)
+            util.deep_setattr(self, name + ".loc", unconstrained_value)
```

Comment:
I'd handle `deep_setattr` as `deep_getattr` and use the `pyro` function, then we don't need to keep them around in the tyxe codebase.